### ISSUE TITLE
Lectern Support for Immersive Book

### ISSUE THREAD

**Issue by hammy275** [hammy3502/immersive-mc#292]

**Comment by hammy275** [comment]
With the OBB support, this would be extremely not-hacky in terms of our rendering and handling. The hack-y part is going to be getting rid of the book that's already there when rendering a lectern normally.

**Comment by hammy275** [comment]
Depends on #386 as well if we want this to be ideal

**Comment by hammy275** [comment]
Still needs to be ported to 1.19.4 and below.

**Comment by hammy275** [comment]
Once I'm done downporting, sometimes the VR->NonVR transition makes a book un-turnable, since the player turning the page is the VR one, so that needs to be fixed.

EDIT: Only happens for some Minecraft versions. 1.19.4 has it, but 1.20.4 doesn't.

**Comment by hammy275** [comment]
Closing this, and moving bugs with it to a separate issue
